libavcodec/vvc/dec.c

```diff
@@ -24,9 +24,15 @@
 #include "libavcodec/decode.h"
 #include "libavcodec/profiles.h"
 #include "libavcodec/refstruct.h"
+#include "libavcodec/aom_film_grain.h"
```

Comment:
maybe we need change this to film_grain.h. 
It's wired to include aom_film_grain.h in vvc